cypress.config.js

```diff
@@ -3,6 +3,22 @@ const { defineConfig } = require('cypress');
 module.exports = defineConfig({
   e2e: {
     setupNodeEvents(on, config) {
+    },
+    baseUrl: 'https://demoqa.com/automation-practice-form',
+    env: {
```

Comment:
you know how to generate random user data - create generateUser function and reuse it in the test